frontend/interfaces/FormTypes.ts

```diff
@@ -0,0 +1,79 @@
+import { UserId } from "./UserTypes";
+
+export interface Form {
+  title: FormTitle;
+  sectionsOrder: SectionId[]; // keeps track of ordering for editing forms
+  sectionsMap: Map<SectionId, FormSection>;
+}
+
+export type FormId = string;
+export type FormTitle = string;
+export type SectionId = string;
+export type FormResponseId = string;
+
+export type FormSection =
```

Comment:
I'm planning for each of the `FormSection` to contain both information about what the question is, as well as the answer the user responds with